test/SPI/private_swiftinterface.swift

```diff
@@ -88,6 +88,20 @@ private class PrivateClassLocal {}
   // CHECK-PUBLIC-NOT: extensionSPIMethod
 }
 
+@_spi(LocalSPI) public protocol SPIProto3 {
+// CHECK-PRIVATE: @_spi(LocalSPI) public protocol SPIProto3
+// CHECK-PUBLIC-NOT: SPIProto3
+
+  associatedtype AssociatedType
+  // CHECK-PRIVATE: associatedtype AssociatedType
+  // CHECK-PRIVATE-NOT: @_spi(LocalSPI) associatedtype AssociatedType
```

Comment:
I'm not sure if this test will fail when you want it to—I'm worried that it could match line 96 against a line like `@_spi(LocalSPI) associatedtype AssociatedType`, and then evaluate line 97 against the rest of the file and decide that the test passes. Can you try running the test against an old compiler and make sure that it fails the way you expect it to?

Reply:
Right, good catch. I've changed it for just a more precise `CHECK-PRIVATE`. If the fix looks good to you I'll PR it for master too.